pkg/internal/config/config.go

```diff
@@ -120,6 +125,7 @@ func New(baseConfig kconfig.Config) (Config, error) {
 		TrafficGeneratorImage:                      TrafficGeneratorImageDefault,
 		VMContainerDiskImage:                       VMContainerDiskImageDefault,
 		TestDuration:                               TestDurationDefault,
+		Verbose:                                    VerboseDefault,
```

Comment:
If the default verbosity is `false`, this initialization is redundant.

Reply:
same answer, making it explicit increased visibility.